### PR TITLE
Auto Player Cycle

### PR DESCRIPTION
**Pull request by stevenschemers**
Added auto player cycle + timeout slider. This allows you to keep your hands on the controller while playing.

### CODE REVIEW

**Comment by stevenschemers** [comment]
Hey, thanks for merging my last PR! I thought of one last feature I think would be great to have. Hope it's not too much trouble. LMK if you have any questions.

File: bakkesmod-playlist-rank-viewer/bakkesmod-playlist-rank-viewer.cpp [hunk 1]
```diff
@@ -29,69 +30,112 @@ void PlaylistRankViewer::onLoad() {
...
-	cvarManager->registerCvar("PRV_show_games_played", "1", "Show games played", true, true, 0, true, 1);
-	cvarManager->registerCvar("PRV_color_rows", "1", "Highlight rows", true, true, 0, true, 1);
-	cvarManager->registerCvar("PRV_background_opacity", "200", "Background opacity", true, true, 0, true, 255);
-	cvarManager->registerCvar("PRV_x_position", "0.800", "Overlay X position", true, true, 0, true, 1.0f);
-	cvarManager->registerCvar("PRV_y_position", "0.080", "Overlay Y position", true, true, 0, true, 1.0f);
-	cvarManager->registerCvar("PRV_scale", "1", "Overlay scale", true, true, 0.5, true, 3);
 
+	/*
+	// Toggle auto cycle (used for keybinding)
+	cvarManager->registerNotifier("PRV_auto_cycle_enabled", [self = this](vector<string>) {
```
Comment (yonilerner) [inline bakkesmod-playlist-rank-viewer/bakkesmod-playlist-rank-viewer.cpp:62]: Is this code used?

Comment (stevenschemers) [reply bakkesmod-playlist-rank-viewer/bakkesmod-playlist-rank-viewer.cpp:62]: Not currently. I was actually hoping you might have some insight on this. I wanted to include a shortcut key to start/stop the auto cycle but for some reason when the dll is first injected during game start the line used to change the cVar value appears to be called which flips the actual value stored in the bakkesmod config.cfg file. 

After the game is loaded the keybinding works as intended. It's just that first load that I could not seem to figure out. Ultimately, if we have to pull it that's fine but the inclusion of the shortcut I think would be a nice addition.

Comment (yonilerner) [reply bakkesmod-playlist-rank-viewer/bakkesmod-playlist-rank-viewer.cpp:62]: Hah yea I had this exact issue with a different setting, I came up with a solution but I forgot what it was. Let me tinker with it and Ill see what I can do. Ill merge your PR and then if I can come up with a solution Ill just do it myself

**Pull request merged**